addons/mail/static/tests/mock_server/mail_mock_server.js

```diff
@@ -178,6 +178,10 @@ async function load_attachments(request) {
     return new mailDataHelpers.Store(IrAttachment.browse(attachmentIds)).get_result();
 }
 
+registerRoute("/web/content/", function () {
+    return;
+});
```

Comment:
Might be worth a comment explaining why it is empty.
Also this should be defined in web and not in mail.